drivers/node/ssh/ssh.go

```diff
@@ -675,6 +676,56 @@ func (s *SSH) SystemCheck(n node.Node, options node.ConnectionOpts) (string, err
 	return file, nil
 }
 
+//GetAvailableDrives returns the block drives on the node
+func (s *SSH) GetAvailableDrives(n node.Node, options node.SystemctlOpts) (map[string]*node.BlockDrive, error) {
+	drives := make(map[string]*node.BlockDrive)
+	driveCmd := fmt.Sprintf("sudo /bin/lsblk -P -s -d -p -o NAME,SIZE,MOUNTPOINT,FSTYPE,TYPE")
+	t := func() (interface{}, bool, error) {
+		out, err := s.doCmd(n, options.ConnectionOpts, driveCmd, false)
+		if err != nil {
+			return out, true, err
+		}
+		return out, false, nil
+	}
+	out, err := task.DoRetryWithTimeout(t, options.ConnectionOpts.Timeout, options.ConnectionOpts.TimeBeforeRetry)
+	if err != nil {
+		return drives, &node.ErrFailedToRunSystemctlOnNode{
```

Comment:
Fix the error here. `ErrFailedToRunSystemctlOnNode` not right ..

Reply:
Fixed